test/ets_hash_ring_test.exs

```diff
@@ -0,0 +1,133 @@
+defmodule ETSHashRingTest do
+  use ExUnit.Case
+  alias HashRingTest.Support.Harness
+
+  setup_all do
+    HashRing.ETS.Config.start_link()
+    :ok
+  end
+
+  setup_all do
+    rings = for num_replicas <- Harness.replicas(), into: %{} do
+      name = :"HashRingETSTest.Replicas#{num_replicas}"
+      {:ok, _pid} = HashRing.ETS.start_link(name, Harness.nodes(), num_replicas)
+      {num_replicas, name}
+    end
+    {:ok, rings: rings}
+  end
+
+  for num_replicas <- Harness.replicas() do
+    describe "ets hash ring, replicas=#{num_replicas}" do
+      for key <- Harness.keys() do
+        test "find_node key=#{key}", %{rings: rings} do
+          assert HashRing.ETS.find_node(rings[unquote(num_replicas)], unquote(key)) == Harness.find_node(unquote(num_replicas), unquote(key))
+        end
+        test "find_nodes key=#{key} num=#{Harness.num()}", %{rings: rings} do
+          assert HashRing.ETS.find_nodes(rings[unquote(num_replicas)], unquote(key), Harness.num()) == Harness.find_nodes(unquote(num_replicas), unquote(key), Harness.num())
+        end
+      end
+    end
+  end
+
```

Comment:
```elixir
alias HashRing.ETS, as: Ring
```

Reply:
✅

Reply:
Where did it go?

Reply:
not sure i follow. its at the top of the file.

Reply:
Did you not push? I'm still seeing `HashRing.ETS...` all over the place.

Reply:
Huh? It's all in this commit: https://github.com/discordapp/ex_hash_ring/commit/cef3241ad7bf32bb8c7298a99fff9f154f04c689

Reply:
I refreshed a bunch of times and didn't see it. What about moving the stetup steps to the setup block?